scripts/west_commands/runners/bossac.py

```diff
@@ -180,9 +181,12 @@ def magic_delay(self):
 
     def make_bossac_cmd(self):
         self.ensure_output('bin')
-        cmd_flash = [self.bossac, '-p', self.port, '-R', '-e', '-w', '-v',
+        cmd_flash = [self.bossac, '-p', self.port, '-R', '-w', '-v',
                      '-b', self.cfg.bin_file]
 
+        if self.erase:
+            cmd_flash += ['-e']
```

Comment:
Would this change making all the boards in tree and out of tree not flash anymore ?

Reply:
They would still flash the given bin, just not erase the entire flash of the target.

Reply:
Perfect!

Please, make sure this is clear on BOSSA docs.

Reply:
I've added a note in the host-tools docs about this. Thanks.